docs/sources/administration/plugin-management/index.md

```diff
@@ -129,7 +136,11 @@ When the update is complete, you see a confirmation message that the uninstall w
 
 ## Install Grafana plugins
 
-Grafana supports data source, panel, and app plugins. Having panels as plugins makes it easy to create and add any kind of panel, to show your data, or improve your favorite dashboards. Apps enable the bundling of data sources, panels, dashboards, and Grafana pages into a cohesive experience.
+Grafana supports data source, panel, and app plugins.
```

Comment:
This whole paragraph feels like unnecessary repetition of content we've covered above, or should have. At this stage it is task based documentation and we shouldn't bloat it.

Reply:
I also agree here as this felt repetition for me too and I also agree that we should simply remove it

Reply:
In my latest commit, I moved to the content up to the top where types were introduced without description, and removed it here. Does that improve?